assets/sass/components/_status-tag.scss

```diff
@@ -0,0 +1,5 @@
+.govuk-tag {
+  &--timeline-tag {
```

Comment:
❤️

Reply:
@pezholio we might want to use this in other places in the future but I didn't want to add it to *all* status tags straight away as it could cause tables to look weird